### DISCUSSION TITLE
Rich text in MessageInput

### DISCUSSION THREAD

**Discussion by EtcGonza** [GetStream/stream-chat-flutter#839] [Ideas]
Hi guys. I thought it would be nice if the MessageInput widget allowed to write rich text. I've been trying really hard to make this feature with the same behavior as the Slack app, but I couldn't.

This is an example of what I was trying to do. 
![image](https://user-images.githubusercontent.com/44512291/147286766-c1774e41-3429-4e9e-9baa-f4a7546ebf62.png)

I think the difficult is that you can write plain text (without rich text) and in some point you if you want to start writing bold text, you can do it without change the text that you write before.
Maybe there is a way to do it, but I couldn't find it. If anyone knows, I would like to read it.

Thank you,
Gonzalo :{D

**Comment by xsahil03x** [comment]
Hey, the MessageWidget uses markdown in order to build the text. So anything which is possible in markdown is possible in Message Widget.

So for a simple strike through text, use can use.

~~My Text~~

Thanks,
Sahil
